sdk/servicebus/service-bus/src/serviceBusError.ts

```diff
@@ -0,0 +1,179 @@
+import { isMessagingError, MessagingErrorCodes, MessagingError, translate } from "@azure/core-amqp";
+import { AmqpError } from "rhea-promise";
+
+/**
+ * Service Bus failure reasons.
+ */
+export type ServiceBusErrorReason =
+  // note: This list is intended to mirror https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusFailureReason.cs
+  /**
+   * The exception was the result of a general error within the client library.
+   */
+  | "GeneralError"
+  /**
+   * A Service Bus resource cannot be found by the Service Bus service.
+   */
+  | "MessagingEntityNotFound"
+  /**
+   * The lock on the message is lost. Callers should call attempt to receive and process the message again.
```

Comment:
```suggestion
   * The lock on the message is lost. Callers should attempt to receive and process the message again.
```
?

Reply:
Yes, fixed as well. I'm guessing it was mean to be "attempt to call" but I like your edit.